cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -57,6 +60,10 @@ const (
 	DefaultEtcdCertDir = "/etc/kubernetes/pki/etcd"
 	// DefaultEtcdClusterServiceName is the default name of the service backing the etcd cluster
 	DefaultEtcdClusterServiceName = "etcd-cluster"
+	// DefaultProxyBindAddressv4 is the default bind address when the advertise address is v4
+	DefaultProxyBindAddressv4 = "0.0.0.0"
+	// DefaultProxyBindAddressv6 is the default bind address when the advertise address is v4
+	DefaultProxyBindAddressv6 = "::"
```

Comment:
I'd like to see the names to be `DefaultProxyBindAddressV4` and `DefaultProxyBindAddressV6`, but it's ok as-is anyway.